src/interpreter.cpp

```diff
@@ -1293,6 +1293,52 @@ ACMD_DECLARE(do_talk);
 ACMD_DECLARE(do_trace);
 ACMD_DECLARE(do_fry_self);
 
+struct command_info mtx_htr_info[] =
+  {
+    { "RESERVED", 0, 0, 0, 0
+    , BLOCKS_IDLE_REWARD },
+    { "afk", 0, do_afk, 0, 0, ALLOWS_IDLE_REWARD },
+    { "disconnect", 0, do_logoff, 0, 1, BLOCKS_IDLE_REWARD },
+    { "logoff", 0, do_logoff, 0, 0, BLOCKS_IDLE_REWARD },
+    { "emote", 0, do_echo, 0, SCMD_EMOTE , BLOCKS_IDLE_REWARD },
```

Comment:
What's the effect of using the emote command while hitching? It just prints to the room you're in, right? Is this for people trying to interact with a hitching character from meatspace, and if so, is the hitcher expected to be able to see/interact with those PCs?

Reply:
It actually doesn't let them emote. If you try to emote  (I tested every command) it just says you can't do that while hitching, because there was an exception already installed in do_emote. Since that code was already written I figured I wouldn't remove it.